ppr-ui/src/components/tombstone/TombstoneDynamic.vue

```diff
@@ -180,14 +179,15 @@ import { useStore } from '@/store/store'
 import { formatExpiryDate, pacificDate } from '@/utils'
 import { RegistrationTypeIF } from '@/interfaces'
 import { MhApiStatusTypes, MhUIStatusTypes } from '@/enums'
-import { useMhrCorrections, useMhrInformation } from '@/composables'
+import { useMhrCorrections, useMhrInformation, useTransportPermits } from '@/composables'
 import { storeToRefs } from 'pinia'
 import { MhrCorrectionClient, MhrCorrectionStaff } from '@/resources'
 import MhrStatusCorrection from '@/components/mhrRegistration/MhrStatusCorrection.vue'
+import { UpdatedBadge } from '../common'
```

Comment:
Will the alias path work here?
`import { UpdatedBadge } from '@/components/common'`

Reply:
Probably, it was auto-imported, will update

Reply:
Done!